README.md

```diff
@@ -10,14 +10,17 @@ Interactive Visualization Platform for Machine Learning and Data Science Dataset
 
 ## Requirements and Setup
 
-In order to run the Neurobazaar Platform you need to have Python and the GCC compiler installed on your computer.
+In order to run the Neurobazaar Platform you need to have Python **3.11** and Node.js **v22.5** installed on your computer.
 
-This software uses VTK version 9.3.1.
+This software uses VTK version **9.3.1**.
 
-This software has been developed and tested with Python **3.11** and GCC **12.3** on Ubuntu 22.04 LTS Server Edition.
+This software has been developed and tested on Ubuntu **22.04 LTS**.
 
 ### Ubuntu Required Packages
 
+To install and manage Node.js installations you can use Node Version Manager. Use the following link to install the latest version of NVM:
+https://github.com/nvm-sh/nvm
```

Comment:
This is not a Ubuntu dependency. Please include the Node.js instructions in a different section. The instructions should be placed after install the Ubuntu dependencies.